posthog/hogql_queries/web_analytics/stats_table.py

```diff
@@ -149,7 +149,7 @@ def to_path_scroll_bounce_query(self) -> ast.SelectQuery:
             min(session.$start_timestamp ) AS start_timestamp
         FROM events
         WHERE and(
-            events.event == '$pageview',
+            or(events.event == '$pageview', events.event == '$screen'),
```

Comment:
Because the front end clearly separates the screen vs. pageview UIs, can we split this in the back end as well? This shouldn't be tricky, we just need to check the same FF for that user, right?